File: course-p-oop/POOUber/Java/UberX.java
package Java;

public class UberX  extends Car{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){
        super(license, driver); //super refer to objects in Car
        this.brand = brand; // this refer to loca objects
        this.model = model;
    }

    
}

File: course-p-oop/POOUber/JS/UberX.js
//Native JS
// function UberX(license, driver, brand, model){
//     Car.call(this, license, driver)
//     this.brand = brand;
//     this.model = model;
// }

//EcmaScript 6
class UberX extends Car{
    constructor(license, driver, brand, model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }
}